istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -86,15 +86,15 @@ func NewCreateRemoteSecretCommand() *cobra.Command {
 		Example: `
 # Create a secret to access cluster c0's apiserver and install it in cluster c1.
 istioctl --Kubeconfig=c0.yaml x create-remote-secret --name c0 \
-    | kubectl -n istio-system --Kubeconfig=c1.yaml apply -f -
+    | kubectl --Kubeconfig=c1.yaml apply -f -
 
 # Delete a secret that was previously installed in c1
 istioctl --Kubeconfig=c0.yaml x create-remote-secret --name c0 \
-    | kubectl -n istio-system --Kubeconfig=c1.yaml delete -f -
+    | kubectl --Kubeconfig=c1.yaml delete -f -
 
 # Create a secret access a remote cluster with an auth plugin
 istioctl --Kubeconfig=c0.yaml x create-remote-secret --name c0 --auth-type=plugin --auth-plugin-name=gcp \
-    | kubectl -n istio-system --Kubeconfig=c1.yaml apply -f -
+    | kubectl --Kubeconfig=c1.yaml apply -f -
```

Comment:
@ayj what would be the cmd if users don't install istio to istio-system ns?

Reply:
Also, where can I find docs for this auth-plugin, auth-type?

Reply:
Yes, user's can use `-n <namespace>` if they install in a different namespace. 

There aren't any additional docs yet for auth->{plugin,type}.